lib/features/pokemon_overview/pokemon_overview_page.dart

```diff
@@ -8,24 +10,38 @@ class PokemonOverviewPage extends StatelessWidget {
     Key? key,
   }) : super(key: key);
 
-  final List<Pokemon> pokemons;
+  final Async<List<Pokemon>> pokemons;
 
   @override
   Widget build(BuildContext context) {
+    void _showErrorMessageSnackbar(String? errorMessage) {
+      final SnackBar snackBar = SnackBar(
+        content: Text(errorMessage!),
```

Comment:
use null coalescing and use emptyString constant as its fallback value instead of using non-null enforce here to be safer 🙂